src/notifications.js

```diff
@@ -12,7 +12,7 @@ class Notifications extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    const {notifications} = nextProps;
+    const notifications = nextProps.notifications.toJS();
```

Comment:
Guys, there not need any changes to use this lib with immutableJS
Diffrence is only in external usage.

Bellow example ...


File with reducers
```javascript 
import {reducer as notificationReducer} from 'react-notification-system-redux';

export function reducer(state = Map({}), action) {
    return Map({
        notifications: notificationReducer(state.get('notifications', []), action),
        // app reducers
        ........
    });
}
```

Usage in container layer
```javascript 
function mapStateToProps(state, props) {
    return {
        notifications: state.get('notifications', [])
    };
}
```
That is it!
Usage in presentation layer has no any difference